docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md

```diff
@@ -1,113 +1,93 @@
 
-*Protip*: Expensify Chat has [moderation tools](https://help.expensify.com/articles/other/Everything-About-Chat#flagging-content-as-offensive) to help flag comments deemed to be spam, inconsiderate, intimidating, bullying, harassment, assault. On any comment, just click the flag icon to moderate conversation. 
+**Post your updates:**
+- Designate a team member to post each update in #announce at the designated time.
+- Each hour, send a message listing exactly what is happening next – if there are multiple sessions happening simultaneously, list out each, along with a description of the session, a reminder of where it's located, and (most importantly) a link to the chat room for that session
+- Write the messages in [markdown format](https://help.expensify.com/articles/other/Everything-About-Chat#how-to-format-text), such that they can be copy/pasted directly into Expensify Chat for sending.
+  - If there is some formatting issue upon posting, no problem: just edit the comment after sending, and it'll be fixed for everyone.
+- We’d also recommend posting your updates on new lines so that if someone has a question about a certain item they can ask in a thread pertaining to that topic, rather than in one consolidated block.
 
-### Step 7: Follow up with attendees after the event
+**Protip**: Your account manager can help you create this document, and would be happy to send each message at the appointed time for you.
 
-Continue the connections by using Expensify Chat to keep your conference community connected. Encourage attendees to share photos, their favorite memories, funny stories, and more. 
+### Step 5: Share Expensify Chat How-To Resources with Speakers, Attendees, Admins
+We’ve created a few helpful best practice docs for your speakers, admins, and attendees to help navigate using Expensify Chat at your event. Feel free to share the links below with them!
 
-- We’d recommend creating a draft of all of your reminders that you plan to send in the #announce (or #social) room throughout the event. 
-- A post in the morning outlining the full agenda, and then before each event as it happens throughout the day, is recommended. Be sure to include details like timings, locations, and any special detail like attire to help attendees feel prepared.
-- Use markdown when posting updates so that your messages are easy to read.
-- We also recommend posting your updates on new lines so that if someone has a question about a certain item they can ask in a thread pertaining to that topic, rather than in one consolidated block. 
+- [Expensify Chat for Conference Attendees](https://help.expensify.com/articles/other/Expensify-Chat-For-Conference-Attendees)
+- [Expensify Chat for Conference Speakers](https://help.expensify.com/articles/other/Expensify-Chat-For-Conference-Speakers)
+- [Expensify Chat for Admins](https://help.expensify.com/articles/other/Expensify-Chat-For-Admins)
```

Comment:
These pages are not ready yet, right?
<img width="1268" alt="image" src="https://github.com/Expensify/App/assets/87341702/c06b5a71-924a-458c-a804-2443958eba69">

Reply:
Out of the scope of this PR, but the not found page looks pretty broken

Reply:
yeah we are adding these in this PR. 

We could probably use relative URLs to make life easier but not a concern atm.